docs/changelog/0032-upgrade-elasticsearch.md

```diff
@@ -0,0 +1,63 @@
+title: ES upgrade from 1.7.6 to 2.4.6
+key: Es-upgrade
+date: 2020-02-05
+optional_per_env: yes
+upgrading_steps: |
+
+  1. stop the site
+
+  commcare-cloud <env> downtime start
+  
+  2. Change the es version and checksum in commcare-cloud/environments/<env>/public.yml 
+  
+  elasticsearch_version: 2.4.6
+  elasticsearch_download_sha256: 5f7e4bb792917bb7ffc2a5f612dfec87416d54563f795d6a70637befef4cfc6f.
+  ELASTICSEARCH_MAJOR_VERSION=2
+
+  3. update the local settings
+  
+  commcare-cloud <env> update-config
+
+  4.login to any ES server and disable the cluster routing
+  
+  curl -XPUT Elasticsearchserver-IP:9200/_cluster/settings -d '{\"transient\" : {\"cluster.routing.allocation.enable\" : \"none\" }}'
```

Comment:
This should be persistent not transient. Transient means it will disappear after restart. I have done this already anyway, but good to keep in the docs here.

Reply:
updated